nsqadmin/options.go

```diff
@@ -6,9 +6,10 @@ import (
 
 type Options struct {
 	LogLevel  string `flag:"log-level"`
-	logLevel  int
 	LogPrefix string `flag:"log-prefix"`
 	Verbose   bool   `flag:"verbose"` // for backwards compatibility
+	Logger    Logger
+	logLevel  int // private, not really an option
```

Comment:
Maybe a bit confusing: I did some moving-around to all three, but then used the new internal shared logging only for nsqd, while experimenting.

Since you approve of this style I'll roll it out to the rest (and we'll see some nice line-count savings).